src/main/java/org/opentripplanner/index/IndexAPI.java

```diff
@@ -79,7 +78,15 @@
 
```

Comment:
The refs above can be removed, and the detail can be moved to the `getAgencyRoutes`method as a parameter

Reply:
I noticed, but I did not want to change too much, but I will do it.